docs/en-us/1.3.6/user_doc/open-api.md

```diff
@@ -1,38 +1,252 @@
- </p>
-
-2. Select the "Expiration time" (Token validity), select "User" (to perform the API operation with the specified user), click "Generate token", copy the Token string, and click "Submit"
-
-<p align="center">
-   <img src="/img/create-token-en1.png" width="80%" />
- </p>
-
-### Use token
-1. Open the API documentation page
-    > Address：http://{api server ip}:12345/dolphinscheduler/doc.html?language=en_US&lang=en
-<p align="center">
-   <img src="/img/api-documentation-en.png" width="80%" />
- </p>
- 
-2. select a test API, the API selected for this test: queryAllProjectList
-    > projects/query-project-list
-                                                                             >
-3. Open Postman, fill in the API address, and enter the Token in Headers, and then send the request to view the result
-    ```
-    token:The Token just generated
-    ```
-<p align="center">
-   <img src="/img/test-api.png" width="80%" />
- </p>  
+# Open API
+
+## Background
+Generally, projects and processes are created through pages, but integration with third-party systems requires API calls to manage projects and workflows.
+
+## Flink Calls Operating steps
+
+### Create queue
+
+1. Log in to the scheduling system, click "Security", then click "Queue manage" on the left, and click "Create queue" to create a queue.
+2. Fill in the name and value of queue, and click "Submit" 
+
+<p align="center">
+   <img src="/img/api/create_queue.png" width="80%" />
+ </p>
+
+
+
+### Create tenant 
+
+```
+1.The tenant corresponds to a Linux user, which the user worker uses to submit jobs. If Linux OS environment does not have this user, the worker will create this user when executing the script.
+2.Both the tenant and the tenant code are unique and cannot be repeated, just like a person has a name and id number.  
+3.After creating a tenant, there will be a folder in the HDFS relevant directory.  
+```
+
+<p align="center">
+   <img src="/img/api/create_tenant.png" width="80%" />
+ </p>
+
+
+
+### Create user
+
+<p align="center">
+   <img src="/img/api/create_user.png" width="80%" />
+ </p>
+
+
+
+### Create token
+
+1. Log in to the scheduling system, click "Security", then click "Token manage" on the left, and click "Create token" to create a token.
+
+<p align="center">
+   <img src="/img/token-management-en.png" width="80%" />
+ </p>
+
+2. Select the "Expiration time" (Token validity), select "User" (to perform the API operation with the specified user), click "Generate token", copy the Token string, and click "Submit"
+
+<p align="center">
+   <img src="/img/create-token-en1.png" width="80%" />
+ </p>
+
+### Use token
+1. Open the API documentation page
+   
+    > Address：http://{api server ip}:12345/dolphinscheduler/doc.html?language=en_US&lang=en
+<p align="center">
+   <img src="/img/api-documentation-en.png" width="80%" />
+ </p>
+
+2. select a test API, the API selected for this test: queryAllProjectList

```

Comment:
It should be a capital S in the beginning.
Instead of Apposition, I think [Select a test API and the API selected for this test is "queryAllProjectList".] could be better.